Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Support setting various labels on the final image
+ARG COMMIT=""
+ARG VERSION=""
+ARG BUILDNUM=""
+
+# Build Geth in a stock Go builder container
+FROM golang:1.22-alpine as builder
+
+# Set the Current Working Directory inside the container
+WORKDIR /story
+
+# Copy go.mod and go.sum files
+COPY go.mod go.sum ./
+
+# Download all dependencies. Dependencies are cached if the go.mod and go.sum files are not changed
+RUN go mod download
+
+ADD . /story/
+RUN go build -o story ./client
+
+# Pull Geth into a second stage deploy alpine container
+FROM alpine:latest
+
+RUN apk add --no-cache ca-certificates
+COPY --from=builder /story/story /usr/local/bin/
+
+EXPOSE 8545 8546 30303 30303/udp
```

Comment:
these ports seem to be the port of `geth`? maybe we need to expose the ports used by `story`

Reply:
It could be. On CL side, 26656 for p2p and 26657 for rpc. In addition, 1317 (by default) for API should be opened for providing API service.

Reply:
I changed expose port. Thanks!